ledger/participant-integration-api/src/main/scala/platform/apiserver/execution/StoreBackedCommandExecutor.scala

```diff
@@ -37,6 +37,8 @@ private[apiserver] final class StoreBackedCommandExecutor(
     packagesService: IndexPackagesService,
     contractStore: ContractStore,
     metrics: Metrics,
+)(implicit
+    ec: ExecutionContext
```

Comment:
can you please add a comment that this is the EC which will execute Engine interpretation (heavy CPU intensive computation)

Reply:
592f65cd2ca08e87e4629a52e4ae28d3701a1f4d